src/mono/mono/mini/aot-compiler.c

```diff
@@ -10733,11 +10733,9 @@ emit_llvm_file (MonoAotCompile *acfg)
 			opts = g_strdup_printf ("%sdefault<O2>,", opts);
 		}
 		opts = g_strdup_printf ("%splace-safepoints' -spp-all-backedges", opts);
-#elif LLVM_API_VERSION >= 1300
+#else
 		/* The safepoints pass requires the old pass manager */
```

Comment:
Didn't we upstream a port of the safe point pass to the new pass manager? I think @jandupej did it, right? Did that make it into 16.0?

Reply:
This is the pre 16.x case, the 16.x case is handled above.

Reply:
@lambdageek Yes, that pass should be available in 16.